go.mod

```diff
@@ -19,6 +19,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require github.com/stretchr/objx v0.4.0 // indirect
```

Comment:
Should this appear in the require list below - before `github.com/stretchr/testify`?

Reply:
Good spot, will update.